packages/create-cloudflare/src/common.ts

```diff
@@ -192,7 +191,7 @@ export const runDeploy = async (ctx: PagesGeneratorContext) => {
 		env: { CLOUDFLARE_ACCOUNT_ID: ctx.account.id, NODE_ENV: "production" },
 		startText: "Deploying your application",
 		doneText: `${brandColor("deployed")} ${dim(
-			`via \`${shellquote.quote(baseDeployCmd)}\``
+			`via \`${baseDeployCmd.join(" ")}\``
```

Comment:
(Related to my more general comment https://github.com/cloudflare/workers-sdk/pull/4445#issuecomment-1814872733)

This is an example of where we still end up using .join(" ") – if command has an values with spaces, this will stringify incorrectly without quotes

Reply:
https://github.com/cloudflare/workers-sdk/pull/4445#discussion_r1396088269